dotCMS/src/main/java/com/dotcms/rest/api/v2/languages/LanguageForm.java

```diff
@@ -0,0 +1,83 @@
+package com.dotcms.rest.api.v2.languages;
+
+import com.dotcms.repackage.javax.validation.constraints.NotNull;
+import com.dotcms.repackage.javax.validation.constraints.Size;
+import com.dotcms.rest.api.Validated;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+@JsonDeserialize(builder = LanguageForm.Builder.class)
+public class LanguageForm extends Validated {
+
+    private final String languageCode;
+
+    private final String countryCode;
+
+    private final String language;
+
+    private final String country;
+
+    public LanguageForm(final LanguageForm.Builder builder) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [It is a good practice to call super() in a constructor](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4616952)